spring-cloud-gcp-dependencies/pom.xml

```diff
@@ -275,6 +275,11 @@
 				<artifactId>cloud-sql-connector-r2dbc-mysql</artifactId>
 				<version>${cloud-sql-socket-factory.version}</version>
 			</dependency>
+			<dependency>
+				<groupId>io.asyncer</groupId>
+				<artifactId>r2dbc-mysql</artifactId>
+				<version>0.9.3</version>
+			</dependency>
```

Comment:
Adding an item in dependencyManagement section does not have any effect. I believe you need to use it in some pom.xml.

Reply:
Did you push the commit?

Reply:
Just did!